src/Management/test/Endpoint.Test/ActuatorRouteBuilderExtensionsTest.cs

```diff
@@ -20,18 +20,31 @@ namespace Steeltoe.Management.Endpoint.Test;
 
 public sealed class ActuatorRouteBuilderExtensionsTest
 {
-    [Fact]
-    public async Task MapTestAuthSuccess()
+
+    public static IEnumerable<object[]> ActuatorOptions
+
+    {
+        get
+        {
+            IHostBuilder hostBuilder = GetHostBuilder(policy => policy.RequireClaim("scope", "actuators.read"));
+            return hostBuilder.Build().Services.GetServices<EndpointOptions>().Select(options => new object[]{options});
+        }
+    }
+    [Theory]
```

Comment:
Changed this to a theory to it is easier to test, for example, if dealing with an issue with a particular actuator we no longer have to set conditional breakpoints etc to figure out what is going on.